spark/src/main/scala/io/ddf/spark/etl/DateUDFs.scala

```diff
@@ -112,6 +112,17 @@ object DateUDF {
     }
   }
 
+  val parseWeekOfMonth: Object => Integer = {
+    (obj: Object) => {
+      val dateTime = Utils.toDateTimeObject(obj)
+      if(dateTime != null) {
+        (dateTime.getDayOfMonth() / 7) + 1
```

Comment:
Good point, this way of computing week of month may not be consistent with the one of weekofyear from the libraries (joda here). A week may span over two months. Also, similar to weekyear, we may need weekmonth too. Currently, we rely on joda to compute those values and joda does not offer weekofmonth. So I think we should not offer this weekofmonth UDF until there are requests from real customers.